f8a_worker/workers/recommender.py

```diff
@@ -607,6 +608,39 @@ def _get_stack_values(self, similar_stacks_list):
             similarity_list.append(s_stack)
         return similarity_list
 
+def apply_license_filter(epv_list):
+    dependencies = []
```

Comment:
you can do `dependencies = licenses = license_score_list = []`

Reply:
DONE. I have clubbed multiple initializations into one line.